generate-specs.js

```diff
@@ -0,0 +1,21 @@
+const $RefParser = require('json-schema-ref-parser')
+const fs = require('fs')
+
+if (!fs.existsSync('./dist')){
+  fs.mkdirSync('./dist');
+}
+
+$RefParser
+  .dereference('./async_api/websocket_api.yaml')
+  .then((schema) => {
+    fs.writeFileSync('dist/async_api.json', JSON.stringify(schema, null, 2))
+  })
+  .then(() => $RefParser.dereference('./open_api/json_api.yaml'))
```

Comment:
Instead of generating a JSON file, could we have written the OpenAPI spec itself in `.json` ? Would that obviate this step?

Reply:
Oh never mind, I believe this package helps with resolving external references too

Reply:
Correct